lintipy.py

```diff
@@ -7,145 +9,47 @@
 import tarfile
 import tempfile
 import time
-from io import BytesIO
-from urllib.parse import urlencode
 
-import boto3
 import jwt
 from botocore.vendored import requests
```

Comment:
You did add `requests` in requirements in this PR. How do you want to go with it? Shall we switch to it, since we are not using boto anyways?